core/domain/exp_domain.py

```diff
@@ -3050,91 +3182,1451 @@ def _convert_states_v51_dict_to_v52_dict(
+                    'buttonText']['value']['unicode_str'] = 'Continuar'
+
+            elif language_code == 'nl':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Doorgaan'
+
+            elif language_code == 'ru':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Продолжить'
+
+            elif language_code == 'sr':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continue'
+
+            elif language_code == 'bg':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continue'
+
+            elif language_code == 'fr':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continuer'
+
+            elif language_code == 'ca':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continua'
+
+            elif language_code == 'hu':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Folytatás'
+
+            elif language_code == 'zh':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = '继续'
+
+            elif language_code == 'it':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continua'
+
+            elif language_code == 'fi':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Jatka'
+
+            elif language_code == 'pt':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continuar'
+
+            elif language_code == 'de':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Fortfahren'
+
+            elif language_code == 'ar':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'استمرار'
+
+            elif language_code == 'cs':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continue'
+
+            elif language_code == 'tr':
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'İlerle'
+
+            else:
+                state_dict['interaction']['customization_args'][
+                    'buttonText']['value']['unicode_str'] = 'Continue'
```

Comment:
This can be done using code to continue dictionary.

Reply:
Hey @vojtechjelinek, I'm not sure what you mean by `code to continue dictionary`. I tried to look into the codebase but not sure where exactly to find it. Can you guide me here?
Thanks

Reply:
you can create a dictionary that maps from language code to the unicode str. Then all this code can be compressed to just a single line of code

Reply:
Done